setup.py

```diff
@@ -243,21 +246,30 @@ def run(self):
             MakePykeRules._pyke_rule_compile()
 
 
+def extract_version():
+    version = None
+    with open('lib/iris/__init__.py') as fd:
```

Comment:
This won't work if setup.py is invoked from a different directory. (But neither will the TestRunner import on the master branch.)

`os.path.join(os.path.dirname(__file__), 'lib/iris/__init__.py')` would be more robust.